scwx-qt/source/scwx/qt/gl/draw/geo_lines.cpp

```diff
@@ -488,23 +493,15 @@ void GeoLines::Impl::UpdateModifiedLineBuffers()
    // Update buffers for modified lines
    for (auto& di : dirtyLines_)
    {
-      // Find modified line in the current list
-      auto it =
-         std::find(currentLineList_.cbegin(), currentLineList_.cend(), di);
-
-      // Ignore invalid lines
-      if (it == currentLineList_.cend())
+      // Check if modified line is in the current list
+      if (currentLineList_.size() < di->lineIndex_ ||
```

Comment:
Should this be `<=`? If the lineIndex is equal to the current size, it's not in the list?